docs/index.rst

```diff
@@ -1,58 +1,22 @@
 
-Welcome to Rikai's documentation!
-=================================
+Welcome to Rikai!
+=================
 
-Rikai is an `Apache Spark`_ based ML data format built for working with
-unstructured data at scale. Processing large amounts of data for ML is never trivial, but that
-is especially true for images and videos often at the core of deep learning applications. We are
-building Rikai with two main goals:
+Rikai is a large-scale data system specicially designed for Machine Learning workflows,
+specialized in Deep Learning development over various forms of unstructured data,
+for example, image, video or sensor data.
 
-1. Enable ML engineers/researchers to have a seamless workflow from feature engineering (`Spark`_)
-   to training (`PyTorch`_/`Tensorflow`_), from notebook to production (:doc:`See details <types>`).
-2. Enable advanced analytics capabilities to support much faster active learning, model debugging,
-   and monitoring in production.
+Rikai strives to offer agnostic developer experice to assist ML engineers at each
```

Comment:
experience

Reply:
what does an "agnostic developer experience" mean?  Usually if you just say "agnostic" without qualifiers, the meaning is religious.

Reply:
`great developer experience`?